### DISCUSSION TITLE
Any ideas for future features? 🤔

### DISCUSSION THREAD

**Discussion by Matek0611** [Matek0611/PospoliteView#4] [New Ideas]
As you can see at the bottom of the [README.md](https://github.com/Matek0611/PospoliteView/blob/main/README.md) file, there are a couple of planned features that I'll try to implement in Pospolite View. In this connection, **do you have any ideas for another features?** If you have something on your mind, don't be shy to share your thoughts! Every little bit helps 😊

**Comment by nomorelogic** [comment]
hi matek0611
Your project is very interesting!
I think a quickstart can be very usefoul in order to start play with your viewer engine.
Few things, just a project and some comments to start.
thanks
nomorelogic

**Comment by nomorelogic** [comment]
Okay, sorry for the misunderstanding
I read that PospoliteView "aims to be a simple HTML5 viewer engine" and the Javascript interpreter is an important planned feature.
What do you think about support for IndexedDB API?
It would be great.
Thank you
nomorelogic

**Comment by Matek0611** [comment]
Hi @nomorelogic,
I think it's a good idea :) I'll consider implementing this feature in next updates.
Matek
